@kiva/kv-components/src/vue/KvLendCta.vue

```diff
@@ -46,8 +46,7 @@
 		<kv-ui-button
 			v-if="isInBasket"
 			variant="secondary"
-			class="tw-inline-flex tw-flex-1"
-			:class="{ 'tw-w-full': showPresetAmounts }"
+			class="tw-inline-flex tw-flex-1 tw-w-full"
```

Comment:
How does this look if there's progress remaining? The issue currently is when a loan is funded, so this solution would work for that case, but otherwise we still show the process bar, right?